pkg/printer/mock_printer.go

```diff
@@ -0,0 +1,17 @@
+package printer
+
+import "github.com/stretchr/testify/mock"
+
+type MockPrinter struct {
```

Comment:
If we have more such instances, we can maybe use mockery to generate mocks against interfaces. Saves the trouble of maintaining boilerplate code.